modules/virt-restoring-vm-from-snapshot-web.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-managing-offline-vm-snapshots.adoc
+
+[id="virt-restoring-vm-from-snapshot-web_{context}"]
+= Restoring a virtual machine from a snapshot in the web console
+
+You can restore a virtual machine (VM) to a previous configuration represented by a snapshot in the web console.
+
+.Procedure
+
+. Click *Workloads* → *Virtualization* from the side menu.
+
+. Click the *Virtual Machines* tab.
```

Comment:
You are correct, but I included this step because it is part of all the other web console procedures in our docs. For example, [Creating virtual machines](https://cnv-7494--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-create-vms.html#virt-creating-vm-wizard-web_virt-create-vms), [Editing virtual machines](https://cnv-7494--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-edit-vms.html#virt-editing-vm-web_virt-edit-vms), etc. 

Maybe leave it there for consistency?

Reply:
Yep, makes sense

Reply:
Thanks!